### PR TITLE
Ability to turn on verbose mode for ssh command.

### LINKED ISSUE #18: Add -v for ssh verbosity
Useful for debugging connection errors. `shipit cp file.txt` isn't working and I don't know why yet :dancers:

### PR DESCRIPTION
**Pull request by Asuza**
Fixes #18.

### CODE REVIEW

File: bin/shipit [hunk 1]
```diff
@@ -62,6 +64,14 @@ version() {
 	echo "`basename $0` v$VERSION"
 }
 
+# Enable verbose mode
+verbose() {
+	echo "$WHITE"
+	echo "Verbose mode is on!"
```
Comment (sapegin) [inline bin/shipit]: I don’t think this message is useful.

File: bin/shipit [hunk 2]
```diff
@@ -194,18 +211,18 @@ copy_to_remote() {
...
 
 # Parse CLI arguments
-arg=$1; shift
-if [ -n "$arg" ]; then
-	case $arg in
+for arg in "$@"; do
+	case "$arg" in
 		-h|--help) usage; exit ;;
 		-V|--version) version; exit ;;
+		-v|--verbose) verbose; ;;
```
Comment (sapegin) [inline bin/shipit]: I’d separate options and commands parsing, fall through here looks unexpected.

Comment (Asuza) [reply bin/shipit]: As in two separate loops? Or something else?

Comment (sapegin) [reply bin/shipit]: Yeah, but you need a loop only for options. There could be only one command.

**Comment by Asuza** [comment]
@sapegin, please review this. I think it might be what you wanted, and it seemed to work in some quick tests, but I'm no bash pro.

**Comment by sapegin** [comment]
Nobody is a Bash pro — if you have less than 50 tabs with Stack Overflow when you write Bash, you’re good enough ;-)

Yep, that’s what I meant. I’m not sure about version and help — they only look like options but behaves like commands but let’s merge it as is for now.

Thank you very much for PRs! 🍕

**Comment by sapegin** [comment]
Out in [0.2.0](https://github.com/sapegin/shipit/releases/tag/0.2.0).

**Pull request merged**